src/libraries/Microsoft.PowerFx.Core/Syntax/IRPreProcessor.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.PowerFx.Syntax
+{
+    internal enum IRPreProcessor
```

Comment:
>IRPreProcessor [](http://example.com/codeflow?start=18&length=14)

We should add a comment for this. 
Xml comments can also refer to specific function, so you can map the